.circleci/config.yml

```diff
@@ -56,6 +64,42 @@ jobs:
       - persist_to_workspace:
           root: ~/repo
           paths: .
+
+  release_rc:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/repo
+
+      # # Download and cache dependencies
```

Comment:
What do you think about removing the commented steps? Was it pushed before? In this case we could retrieve from earlier versions in case we need this in the future. But I am also ok to keep it. They are not impacting much readability.

Reply:
Agree. Solved!

Reply:
Nice! Thanks.